src/Config/XmlConfigFileLoader.php

```diff
@@ -0,0 +1,220 @@
+<?php declare(strict_types=1);
+
+
+namespace Shopware\Psh\Config;
+
+use DOMElement;
+use DOMXPath;
+use Symfony\Component\Config\Util\XmlUtils;
+
+/**
+ * Load the config data from an xml file
+ */
+class XmlConfigFileLoader extends ConfigFileLoader
+{
+    const NODE_HEADER = 'header';
+
+    const NODE_PLACEHOLDER = 'placeholder';
+
+    const NODE_PLACEHOLDER_DYNAMIC = 'dynamic';
+
+    const NODE_PLACEHOLDER_CONST = 'const';
+
+    const NODE_PATH = 'path';
+
+    const NODE_ENVIRONMENT = 'environment';
+
+    const NODE_TEMPLATE = 'template';
+
+    const NODE_TEMPLATE_SOURCE = 'source';
+
+    const NODE_TEMPLATE_DESTINATION = 'destination';
+
+    /**
+     * @var ConfigBuilder
+     */
+    private $configBuilder;
+
+    /**
+     * @var string
+     */
+    private $applicationRootDirectory;
+
+    /**
+     * @param ConfigBuilder $configBuilder
+     * @param string $applicationRootDirectory
+     */
```

Comment:
Der php doc block kann von mir aus weg. Steht ja alles schon in der Methoden Signatur drin :-)